concordia/static/js/contribute.js

```diff
@@ -429,13 +431,33 @@ var hideTooltipCallback = function() {
     };
     setTimeout(hideTooltip($(this)), 3000);
 };
+
+function trackShareInteraction(element, interactionType) {
+    // Adobe analytics user interaction tracking
+    if (window['loc_ux_tracking'] != undefined) {
+        let loc_ux_tracking = window['loc_ux_tracking'];
+        loc_ux_tracking.trackUserInteractionEvent(
+            element,
```

Comment:
Does this expect to get jQuery objects or DOM elements? The variable is named `element` but the values being passed in are jQuery objects. If that's correct, I'd like to rename this to something more obvious like `$element` to make it clear that it's not a normal `Element`.

Reply:
It expects jQuery objects. Changed it.